open-apparel-registry/open-apparel-registry
# Update spelling of TR to Türkiye and accept multiple versions

## Overview

The official spelling of Turkey has been updated by the Turkish Government to "Türkiye"

In addition to changing how country code TR is displayed we update the parsing helper function to acceppt multiple spellings with different puncuation.

Connects #2363

## Demo

### Updated name in country dropdown

<img width="270" alt="Screenshot 2023-03-01 at 11 43 55 AM" src="https://user-images.githubusercontent.com/17363/222237936-e7ebf07b-9d7c-4f1d-9b45-cfa8a9397c22.png">

### Updated country name returned from the API

<img width="742" alt="Screenshot 2023-03-01 at 11 45 10 AM" src="https://user-images.githubusercontent.com/17363/222238054-4066598f-1228-43b1-98d3-cc7004b3a48a.png">

### List parsing handles multiple variations

<img width="549" alt="Screenshot 2023-03-01 at 11 50 15 AM" src="https://user-images.githubusercontent.com/17363/222239169-7ada9c0a-e047-451f-8e20-665cd93864fd.png">

## Testing Instructions

* Log in as c3@example.com
* Upload [tr-test.csv](https://github.com/open-apparel-registry/open-apparel-registry/files/10864476/tr-test.csv)
* Run `./scripts/manage batch_process -a parse -l {list id}`
* Verify that all of the rows parse correctly and show "Türkiye"

## Checklist
